presto-main/src/main/java/io/prestosql/sql/planner/planprinter/TextRenderer.java

```diff
@@ -400,6 +400,13 @@ public String detailIndent()
             return nextLinesPrefix + pad(indent, 4);
         }
 
+        /**
+         * Adds spaces at the end of a string until it reaches a specific length.
+         *
```

Comment:
Shouldn't we add a description explaining why this padding may be useful for presto?

Reply:
added